src/technologydata/constants/energy_density.py

```diff
@@ -26,36 +26,278 @@
+                    url="https://github.com/kyleniemeyer/computational-thermo",
+                ),
+            ],
+        ),
+    ),
+    wood=Parameter(
+        magnitude=15.4,
+        units="GJ/t",
+        carrier="wood",
+        sources=SourceCollection(
+            sources=[
+                Source(
+                    title="The Engineering ToolBox",
+                    authors="The Engineering ToolBox (2003). Higher Calorific Values of Common Fuels: Reference & Data. [online] Available at:https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html [Accessed 6 September 2025].",
+                    url="https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html",
+                ),
+            ],
+        ),
+    ),
+    carbon=Parameter(
+        magnitude=32.8,
+        units="GJ/t",
+        carrier="carbon",
+        sources=SourceCollection(
+            sources=[
+                Source(
+                    title="The Engineering ToolBox",
+                    authors="The Engineering ToolBox (2003). Higher Calorific Values of Common Fuels: Reference & Data. [online] Available at:https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html [Accessed 6 September 2025].",
+                    url="https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html",
+                ),
+            ],
+        ),
+    ),
+    methanol=Parameter(
+        magnitude=19.9,
+        units="GJ/t",
+        carrier="methanol",
+        sources=SourceCollection(
+            sources=[
+                Source(
+                    title="The Engineering ToolBox",
+                    authors="The Engineering ToolBox (2003). Higher Calorific Values of Common Fuels: Reference & Data. [online] Available at:https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html [Accessed 6 September 2025].",
+                    url="https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html",
+                ),
+            ],
+        ),
+    ),
+    gasoline=Parameter(
+        magnitude=43.4,
+        units="GJ/t",
+        carrier="gasoline",
+        sources=SourceCollection(
+            sources=[
+                Source(
+                    title="The Engineering ToolBox",
+                    authors="The Engineering ToolBox (2003). Higher Calorific Values of Common Fuels: Reference & Data. [online] Available at:https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html [Accessed 6 September 2025].",
+                    url="https://www.engineeringtoolbox.com/fuels-higher-calorific-values-d_169.html",
+                ),
+            ],
+        ),
     ),
     # Add more energy densities as needed
 )
 
 EnergyDensityHHV: dict[str, Parameter] = dict(
     hydrogen=Parameter(
         magnitude=141.8,
```

Comment:
```suggestion
        magnitude=141.7,
```

At least that is what I see when looking at the table